core/src/main/kotlin/Formats/JavaLayoutHtml/JavaLayoutHtmlFormatOutputBuilder.kt

```diff
@@ -767,6 +770,30 @@ open class JavaLayoutHtmlFormatOutputBuilder(
         return false
     }
 
+    protected open fun FlowContent.apiAndDeprecatedVersions(node: DocumentationNode) {
+        val apiLevelExists = node.apiLevel.name.isNotEmpty()
+        val deprecatedLevelExists = node.deprecatedLevel.name.isNotEmpty()
+        if (apiLevelExists || deprecatedLevelExists) {
+            div(classes = "api-level") {
+                if (apiLevelExists) {
+                    +"added in "
```

Comment:
added => Added

Reply:
Thanks! I'll change it :) I thought it was an intentional change to the Javadoc version